### DISCUSSION TITLE
I wanna make a mod but how do I get da code

### DISCUSSION THREAD

**Discussion by NATIONALNANAYT** [FunkinCrew/Funkin#4048] [Q&A] [answered]
Please help

**Comment by JackXson-Real** [comment]
If you arent able to get the code then I doubt you can make a mod

**Comment by moxie-coder** [comment]
or make one at all if you're literally gonna make an issue about it

**Comment by NATIONALNANAYT** [comment]
Jezz dude

**Comment by amyspark-ng** [comment, marked as answer]
you don't necessarily need code there's this person on youtube that's teaching how to make mods in fnf v-slice
https://www.youtube.com/@catbrotherdoeseverything
if you need something more complex, i don't really know
